README.md

```diff
@@ -26,6 +26,9 @@ These values are placed in the conf.py of your sphinx project.
     * This is not required. Name of the site. This is displayed above the title.
 * `ogp_image`
     * This is not required. Link to image to show.
+* `ogp_image_alt`
+    * This is not required. Alt text for image. Defaults to using `ogp_site_name` or the document's title as alt text,
+    if available. Set to `False` if you want to turn off alt text completely.
```

Comment:
Can you put this all on a a single line, to keep consistent with the rest of the readme? (Yeah I know it breaks PEP)

Reply:
No problem!